source/PlayerInfo.cpp

```diff
@@ -1484,7 +1484,8 @@ void PlayerInfo::Land(UI *ui)
 	if(!freshlyLoaded)
 	{
 		Audio::Play(Audio::Get("landing"));
-		Audio::PlayMusic(planet->MusicName());
+		if(!planet->MusicName().empty())
+			Audio::PlayMusic(planet->MusicName());
```

Comment:
Is this not made redundant by the first part of Audio::UpdateMusic?

Reply:
yes